openc3-cosmos-script-runner-api/Dockerfile

```diff
@@ -21,6 +21,7 @@ RUN bundle config set --local without 'development' \
 RUN ["chown", "-R", "openc3:openc3", "/src/"]
 COPY  --chown=${IMAGE_USER}:${IMAGE_GROUP} ./ ./
 RUN ["chmod", "-R", "777", "/src/"]
+RUN ["chmod", "-R", "555", "/src/scripts"]
```

Comment:
Looks like `running_script` sets the process cwd to `/src/scripts` and we execute out of there. Can you document why this is needed or is more secure?